bean-consistency-checker/src/test/java/com/redhat/lightblue/migrator/consistency/BeanConsistencyCheckerTest.java

```diff
@@ -64,6 +66,29 @@ public void testInconsistencyList() {
     }
 
     @Test
+    public void testConsistencyArray() {
+        Country[] cArr1 = new Country[] { new Country(1l, "PL"), new Country(2l, "CA")};
+        Country[] cArr2 = new Country[] { new Country(1l, "PL"), new Country(2l, "CA")};
+
+        Assert.assertTrue(Objects.deepEquals(cArr1, cArr2));
+        // we need to use deepEquals for arrays
+        Assert.assertFalse(Objects.equals(cArr1, cArr2));
+    }
+
+    @Test
+    public void testInconsistencyArray() {
+        Country[] cArr1 = new Country[] { new Country(1l, "PL"), new Country(2l, "CA")};
+        Country[] cArr2 = new Country[] { new Country(3l, "PL"), new Country(2l, "CA")};
+
+        Assert.assertFalse(Objects.deepEquals(cArr1, cArr2));
+
+        Country[] cArr3 = new Country[] { new Country(2l, "CA"), new Country(1l, "PL")};
+        Country[] cArr4 = new Country[] { new Country(1l, "PL"), new Country(2l, "CA")};
+
+        Assert.assertFalse(Objects.deepEquals(cArr3, cArr4));
+    }
```

Comment:
The point is to test beanConsistencyChecker.consistent method, not Objects.deepEquals.